modules/candidate_list/help/candidate_list.md

```diff
@@ -1,66 +1,7 @@
-"Basic" filter options. Users may select from drop-down select
-boxes, and then click "Show Data" to view a list of candidates. If
-only a fragment of the candidate ID is known, the user may search
-for the profile by entering the known fragment, and then click show
-data to find the appropriate profile.
-
-For example, searching with "531" under "DCCID" will retrieve the
-profile of candidate DCC1107 (with full DCCID 531861).
-
-## Option 3: Advanced Filter Options
-
-More detailed filter options can be accessed by clicking on the
-"Advanced" button. Users will select from a number of drop-down
-select boxes including Site, Project, Subproject, Biological Sex,
-etc., and then click "Show Data" to view a list of candidates. To
-return to "Basic" selection filters please select "Basic".
-
-To view a specific candidate's dataset, click on the candidate's
-PSCID found in blue text under the "PSCID" column. At the top of
-the table, column headings will appear underlined and show a click
-icon when the user hovers over the heading title. Data can be sorted
-in ascending order according to a given column by clicking on the
-column heading (i.e. DCCID, Site, PSCID etc.), and by clicking
-again, in descending order. In both standard and mobile view, the
-selection filter can be hidden by selecting the upward arrow icon,
-allowing easier viewing of the data. Once in this view, users can
-click on the downward arrow icon to display the features of the
-selection filter once again. This feature applies to all pages that
-include a Selection Filter.
-
-Note that the form remembers previously selected data such that
-when the user returns to the "Access Profile" page, the selection
-filter will automatically select profiles according to the last
-selection settings. Depending on the magnitude of the search, there
-may be several pages of search results for the user to look through.
-If "No candidates found" appears, this indicates no profiles matched
-the information specified by the selection filter(s).
-
-When on a mobile device or narrow browser window, the table will
-have a slider at the bottom of the table, facilitating scrolling
-of the data. The user may also hover or click the arrows that flank
-the table to see the remainder of the table. When scrolling through
-search results, the PSCID column will freeze and remain displayed
-on the lefthand side, to allow for easy reference and access to
-relevant information.
+You can search for a specific candidate using any of the filter options in the *Selection Filter* section. If necessary, click **Show Advanced Filters** to search using additional filters. Your matching candidate(s) will appear automatically in the table below as you populate the filter fields. Depending on how many filters you use, you may have multiple candidates appear in the table. For example, if you only enter a DCCID, any candidate with that ID will appear in the table. 
```

Comment:
Nice.  since `DCCID` is unique to an individual - try Site ?   
Also, partial matches are important to mention: If you enter a partial PSCID e.g. `MTL001` you'll get matched to MTL0010 and MTL0011, and so on

Reply:
done (both)

Reply:
spotted an error:  it's not possible for `MTL001` to be a `DCCID` (which is always 6 random digits) -> i'll fix this with a commit.